README.md

```diff
@@ -135,14 +135,13 @@ Example command:
 
```

Comment:
- [ ] OPTIONAL
Kindly add instructions in your ReadMe under the _Getting Started_ section around _line 59_, on how to run the test cases of your project.